serialzy/serializers/tensorflow.py

```diff
@@ -62,4 +78,14 @@ def _deserialize(self, source: BinaryIO, schema_type: Type, user_type: Optional[
 
     def _types_filter(self, typ: Type):
         import tensorflow as tf  # type: ignore
-        return typ in [tf.train.Checkpoint] or (inspect.isclass(typ) and issubclass(typ, tf.Module))
+        if typ in [tf.train.Checkpoint]:
+            return True
+        if inspect.isclass(typ) and issubclass(typ, tf.Module):
+            return True
+        return self._is_generic_user_object(typ)
+
+    @staticmethod
+    def _is_generic_user_object(typ: Type) -> bool:
+        return inspect.isclass(type)\
+               and typ.__module__ == "tensorflow.python.saved_model.load"\
```

Comment:
looks nasty

Reply:
also, im not 100% sure that any class object will have `__module__` attribute, maybe you should use `getattr(typ, '__module__', '') == "tensorflow.python.saved_model.load"` for better safety instead